.github/workflows/ci.yml

```diff
@@ -26,3 +27,85 @@ jobs:
     - run: make lint
     - run: make test
 
+  docker:
```

Comment:
`docker` feels a bit ambiguous (do *what* with docker?).
How about `build` or `build-and-push`?